### PR TITLE
Add more prometheus labels from vespa metric dimensions

### PR DESCRIPTION
**Pull request by ghost**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by dkurzaj** [comment]
I was just wondering (I haven't touched the exporter for a while): why do we limit the types of dimensions we want as labels?

**Comment by ghost** [comment]
Ideally creating labels should have been dynamic in the prometheus-client. In that way we could
have added labels to a metric on the fly.

As far as I know this is not the case, and they have to be specified at metric creation.

File: vespa-exporter.py [hunk 1]
```diff
@@ -180,7 +180,7 @@ def get_container_metrics(hostport):
             labelvalues = {}
             labelvalues['host'] = hostport
             if 'dimensions' in v:
-                for d in ['chain', 'handler', 'api', 'operation', 'status', 'serverName', 'serverPort']:
+                for d in ['chain', 'handler', 'api', 'operation', 'status', 'serverName', 'serverPort', 'httpMethod', 'gcName']:
```
Comment (dkurzaj) [inline vespa-exporter.py]: I meant, why don't we also add here for example: `documenttype`, `field`, `rankProfile` or `docidPartition`? (those are just some dimensions I stumbled upon)

Comment (ghost) [reply vespa-exporter.py]: oh, I see. Good point. 

I cannot remember why I limited the dimensions to this short list. I guess I was just so overwhelmed by the amount of metrics. 

There is probably no reason why we cant just add all found dimensions.

Comment (dkurzaj) [reply vespa-exporter.py]: Ok, I did not find more than those four for now, can you maybe add them in this PR? And others you might find too! :)

**Comment by dkurzaj** [comment]
Thank you! :)

LGTM

**Pull request merged**